packages/db/src/lib/execution.ts

```diff
@@ -12,6 +12,34 @@ export interface arnRecord {
 
 const log = new Logger({ sender: '@cumulus/db/lib/execution' });
 
+/**
+ * Returns a list of executionArns sorted by most recent first, for an input
+ * Granule Cumulus ID.
+ *
+ * @param {Knex | Knex.Transaction} knexOrTransaction
+ *   Knex client for reading from RDS database
+ * @param {number} granuleCumulusId - The primary ID for a Granule
+ * @returns {Promise<arnRecord[]>} - Array of arn objects with the most recent first.
+ */
+export const getExecutionArnsByGranuleCumulusId = (
+  knexOrTransaction: Knex | Knex.Transaction,
+  granuleCumulusId: Number
+): Promise<arnRecord[]> =>
+  knexOrTransaction(tableNames.executions)
+    .select('arn')
```

Comment:
Interesting that we don't need a specific table namespace for this.  :thinking:

Reply:
Hmm yeah. I see your point 😄 

The correct way to do it should be something like `${tableNames.executions{.arn}` but maybe Knex magics this away and assumes the "FROM" table unless specified otherwise? I'll update it anyway. We should be clearer where the column is coming from.

Reply:
https://github.com/nasa/cumulus/pull/2415/commits/eb8507804e1276056572a08ca5e978a50e3724e0